### DISCUSSION TITLE
I want to renew my student account, but when I try to verify my school, the page does not advance

### DISCUSSION THREAD

**Discussion by emreihtiyar** [community/community#136486] [GitHub Education] [answered]
### Hi everyone,

Hello, I want to renew my student registration but after selecting my school it gets stuck when I want to continue. you can see it in the attached video. 

https://github.com/user-attachments/assets/fe62bb04-fa62-4b20-8015-99ee6ae5766c

When I look at the Network tab of my browser, 

status code 200 is returned to my request, but the next page does not load. I waited for a few minutes in case it was still loading but unfortunately it didn't load.

**Comment by davevad93** [comment]
Hi @emreihtiyar ,

Have you tried either clearing the cache or using another browser? It could be due to that.

**Reply by emreihtiyar** [reply, marked as answer]
Yes, I tried several times on both chromium and Safari but it didn't work. I also tried incognito and clearing cache but to no avail. 
When I tried on my mobile device the problem was solved. 

Thank you

**Reply by davevad93** [reply]
Glad you solved it man! 👍
